src/main/resources/diagram_git_starship.puml

```diff
@@ -2,132 +2,181 @@
 
 package rape.brutal.gitstarship {
 
-    package ships {
+    class Universe {
+        -int kills;
+        -int moneySpends;
+        +void startUniverse();
+        +void winState();
+        +void loseState();
+        +void getStats();
+    }
 
-        Starship <|-- GitStarship
-        Starship <|-- PirateStarship
+    class Storage {
+        -HashMap<String, IStorageable> items;
+        +void putItem(IStorageable iStorageable);
+        +void getItem(String itemName);
+        -int capacity;
+    }
 
-        abstract class Starship {
-            #Engine engine;
-            #Hull hull;
-            #Storage<IStoreable> storage;
+    package trade {
 
-            +boolean putInStorage(IStoreable iStoreable);
-            +IStoreable getFromStorage(String iStoreableName);
+        abstract class Trader {
+            -int cash;
+            -Storage storage;
+        }
 
-            -void flyTo(IReachable iReachable);
+        class AverageTrader extends Trader {
+            +void sell(int cash, String itemName);
+            +void buy(int cash, String itemName);
+        }
 
-            +void trade(Trader trader);
+        class Mechanic extends Trader {
+            +void repair(int cash, String itemName);
+            +void upgrade(int cash, String itemName);
+        }
 
-            +void setDamage(int damage);
-            +int getHitPoints();
+    }
 
-            +void attack(IAttackable iAttackable);
-        }
+    package events {
 
-        class GitStarship {
-            -Railgun railgun;
-            -Turret rightTurret;
-            -Turret leftTurret;
+        +abstract class Event {
+            #Universe universe;
+            #Location location;
+            #GitStarship gitStarship;
+
+            +void start(GitStarship gitStarship);
         }
 
-        class PirateStarship {
-            -Turret rightTurret;
+        +class Battle {
```

Comment:
Very stupid class.

Possible NPE for `gitStarship`, NPE for `universe` etc.

What is purpose of this class?